js/form.js

```diff
@@ -21,19 +27,12 @@
   window.form.toggleFormNodesState(true);
 
   var onFormChange = function () {
-
-    var ROOMS_FOR_GUESTS_MAP = {
-      '1': ['1'],
-      '2': ['1', '2'],
-      '3': ['1', '2', '3'],
-      '100': ['0']
+    var roomsValidity = {
+      term: ROOMS_FOR_GUESTS_MAP[adFormNode.rooms.value].includes(adFormNode.capacity.value),
```

Comment:
Тут у нас хранится булево значение, лучше переименовать:
```suggestion
      isValid: ROOMS_FOR_GUESTS_MAP[adFormNode.rooms.value].includes(adFormNode.capacity.value),
```